requirements.txt

```diff
@@ -0,0 +1,8 @@
+xmldiff
+yamlpath
+json2html
+types-openpyxl
+types-PyYAML
+openpyxl-stubs
+cmakelint
+-r docs/requirements.txt
```

Comment:
We'll also need:

```
$ ./utils/ansible_playbook_to_role.py                                                                                                                                                                                                                                                                                                                                                                                     Please install PyGithub, on Fedora it's in the python-PyGithub package.
```

Reply:
That text should be changed to `python3-PyGithub` that package does exist in Fedora.